ci-scripts/unit-tests-sonar.sh

```diff
@@ -21,6 +21,15 @@ if [[ -n "$coverage" ]]; then
     exit 1
 fi
 
+echo "Running unit tests and code coverage for vendor"
```

Comment:
Vendor libraries should have their own scripts to run their own tests. We won't be doing that as part of ours.